index.d.ts

```diff
@@ -1,44 +1,43 @@
-/// <reference types="node"/>
-import {ChildProcess} from 'child_process';
-import {Stream, Readable as ReadableStream} from 'stream';
-
-declare namespace execa {
-	type StdioOption =
-		| 'pipe'
-		| 'ipc'
-		| 'ignore'
-		| 'inherit'
-		| Stream
-		| number
-		| undefined;
-
-	interface CommonOptions<EncodingType> {
-		/**
+import {Buffer} from 'node:buffer';
+import {ChildProcess} from 'node:child_process';
+import {Stream, Readable as ReadableStream} from 'node:stream';
+
+export type StdioOption =
+	| 'pipe'
+	| 'ipc'
+	| 'ignore'
+	| 'inherit'
+	| Stream
+	| number
+	| undefined;
+
+export interface CommonOptions<EncodingType> {
+	/**
 		Kill the spawned process when the parent process exits unless either:
 			- the spawned process is [`detached`](https://nodejs.org/api/child_process.html#child_process_options_detached)
 			- the parent process is terminated abruptly, for example, with `SIGKILL` as opposed to `SIGTERM` or a normal exit
 
 		@default true
 		*/
-		readonly cleanup?: boolean;
+	readonly cleanup?: boolean;
 
-		/**
```

Comment:
Would it be possible to separate all whitespace changes to a separate PR?
This will make reviewing this PR easier. Thanks!

Reply:
I think I had to update `xo` for ESM (see other PRs by Sindre where he updated his dependencies). Don't remember the exact reasons though. I'd recommend reviewing with `?w=1` in the URL, it helps a lot.

It's a bit tricky to not have indentation changes because I had to "unwrap" definitions and hoist them to the top level. Maybe I can just apply Prettier in a separate commit, and then my code changes on top?

Reply:
Yes, I understand this might be a little of a chicken-and-egg problem there.
If Sindre is doing it like this on other repositories, let's be consistent and update both at once :+1:

Reply:
Actually this is really bugging me, so I'll take another stab at reducing the whitespace diff (similar to https://github.com/sindresorhus/is-stream/commit/5831295859122e5a246f31d6b92c8d5e8b551d5e?w=1)